tests/utils/queryApiGateway.test.ts

```diff
@@ -107,17 +107,16 @@ describe('Querying the API Gateway', () => {
       status: 403,
     })
     /* eslint-enable  @typescript-eslint/no-unsafe-assignment */
-    fetch.mockRejectedValue(errorResponse403)
+    fetch.mockResolvedValue(errorResponse403)
```

Comment:
Note: I don't have a great grasp of `fetch`, but I believe the issue here is that a 403 response isn't actually a rejected promise. It's a resolved promise that contains the 403 status code.

Reply:
that makes sense to me!